data/sql_updates/create_omnibus_dates.sql

```diff
@@ -58,8 +58,7 @@ $$ language plpgsql;
 
 
 -- Not all report types are on dimreporttype, so for the reports to all have
--- titles, I am adding a case. Ideally, we would want the right mapping,
--- that is one of the things we have asked for.
+-- titles, I am adding a case. Ideally, we would want the right mapping.
 create or replace function name_reports(office_sought text, report_type text, rpt_tp_desc text, election_state text[])
```

Comment:
Yes, sometimes I found that was not populated so I show the code if we can't identify it.